Changelog.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `VK_KHR_get_display_properties2` instance extension (#932)
 - Added `VK_EXT_metal_objects` device extension (#942)
 - Added `VK_AMD_anti_lag` device extension (#943)
+- Added `VK_KHR_video_queue`, `VK_KHR_video_encode_queue`, and `VK_KHR_video_decode_queue` device and instance extensions (#965)
```

Comment:
All of these extensions are still regarded as "device extensions" by the documentation (https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_video_queue.html, https://registry.khronos.org/vulkan/specs/latest/man/html/VK_KHR_video_encode_queue.html). The fact that the `PhysicalDevice` query exists on the `::Instance` is wrapper (using an instance function, rather than a device-specific function) doesn't make them instance extensions (but it does blur the lines/meaning somewhat).